ItemLevelDownloader/Engine/artifactEngine.ts

```diff
@@ -6,11 +6,13 @@ var minimatch = require('minimatch');
 import * as models from '../Models';
 import { ArtifactItemStore } from '../Store/artifactItemStore';
 import { ArtifactEngineOptions } from "./artifactEngineOptions"
+import { Logger } from './logger';
 
 export class ArtifactEngine {
     async processItems(sourceProvider: models.IArtifactProvider, destProvider: models.IArtifactProvider, artifactEngineOptions?: ArtifactEngineOptions): Promise<void> {
         const processors: Promise<{}>[] = [];
         artifactEngineOptions = artifactEngineOptions || new ArtifactEngineOptions();
+        this.logger = new Logger(artifactEngineOptions.verbose);
```

Comment:
is artifactEngineOptions.verbose determined by System.Debug variable?

Reply:
yes, it should be, tasks need to have that logic.